spectator-web/src/main/java/com/netflix/spectator/controllers/model/MetricValuesMap.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spectator.controllers.model;
+
+import java.util.HashMap;
+
+/**
+ * Maps a metric name to its values.
+ */
+public class MetricValuesMap extends HashMap<String, MetricValues> {
```

Comment:
Why not just use `HashMap<String, MetricValues>`? This class doesn't seem to add anything.